adam/batch.py

```diff
@@ -185,6 +214,11 @@ def generate_opm(self):
         Returns:
             OPM (str)
         """
+
+        # State vector is required in the OPM even if keplerian elements are also given. However,
+        # in that case it will be ignored in favor of the keplerian elements so it is not required
```

Comment:
Do we really still require state vector even if keplerian elements are given? Should probably fix it at some point, not necessarily now.

Reply:
I don't think we do, but OPM standard does. So this lets user specify one or the other but still fills in opm with dummy values as required.